### ISSUE TITLE
[Feature Request] Remember favorite filter per tab

### ISSUE THREAD

**Issue by Ammoton** [jmshrv/finamp#558]
Hello, I personally have an issue with songs featuring multiple artists. I prefer not to tag them with only one artist, but if I don't, the artist view becomes cluttered with many artists having only one song.

To address this, I appreciate the feature that allows showing only favorite artists with the heart icon. However, I would prefer this option to apply only to artists, excluding albums and songs. This setting can be included in the options for a cleaner interface.

Option: "Show only favorite artists" (Yes/No)
Additionally, include the option to display a heart icon, enabling users to toggle between showing all artists and displaying only their favorites.

**Comment by Chaphasilor** [comment]
We could also instead only show album artists, that should at least get rid of some featured artists?   

And in principle, having the "favorites only" filter separately for all tabs shouldn't be a huge problem either...

**Comment by Ammoton** [comment]
Yes, that's a great idea, and having the "favorites only" filter separately for all tabs would be very helpful.

**Comment by mihawk90** [comment]
I'm late to the party but I just noticed this issue scrolling through the list on the search for something else...

Reading what you're describing actually makes me wonder how your music is tagged and if this is even an issue that Finamp can solve, or if that's something happening on the Jellyfin side.

Just as a recent example for me, I have this album:
https://musicbrainz.org/release/281334e1-5df2-47d9-915b-0c58adfc8dca

As you can see most of the tracks have multiple artists and they are tagged as such in my files and in Jellyfin. However, the entire album only shows up under `ilan Bluestone`, the album artist.
On the Jellyfin side only `ilan Bluestone` is listed in the album artists as well. `Giuseppe de Luca` for example shows up under artists, but not in Finamp (except on the track-listing and now playing screen of course).
The artists are tagged as a multi-value field:
```
mediainfo 01\ Scars\ -\ ilan\ Bluestone\ feat.\ Giuseppe\ de\ Luca.flac | grep -i artist
ALBUMARTISTSORT                          : Bluestone, Ilan
ARTISTSORT                               : Bluestone, Ilan feat. de Luca, Giuseppe
ARTISTS                                  : ilan Bluestone / Giuseppe de Luca
```
Note the `/` is inserted by `mediainfo`, technically these are 2 fields in the tag-data (here in kid3):
![image](https://github.com/jmshrv/finamp/assets/556162/d8112454-4e23-4eed-80d5-c78cc25bf207)

Now granted these files are FLAC (using VorbisComment), but MP3s have had the same ability since ID3v2.4, i.e. since 2000.
For ID3v2.3 (and below(?)) Jellyfin manually splits the strings, and I believe the default field-delimiter is a semi-colon.

Either way, what I'm saying is I'm wondering how this looks on the Jellyfin side in the webinterface.

**Comment by Ammoton** [comment]
Yes, you are right. My music was not tagged correctly. I use Audials to record my music, and the program automatically tags all artists as album artists. 

(I noticed this recently but forgot about the issue.)

@Chaphasilor should I close the issue or leave it open because of the "favorites only" filter separately for all tabs?

**Comment by Chaphasilor** [comment]
Yeah I also noticed that Finamp already only shows album artists. Jellyfin has the ability to show artists without any albums assigned, but showing those in Finamp makes little sense until #572 has been implemented.

Keep the issue open, I'll rename it for now :)

**Comment by Ammoton** [comment]
@Chaphasilor After taking another look, I don't find it very useful anymore, and it could even make things more confusing if it's different per tab. With the beta redesign moving away from tab navigation anyway, this may not be as relevant.

Would it be alright if I close this issue to declutter the issue tracker?

**Comment by Chaphasilor** [comment]
@Ammoton sure, it's always nice to close some issues.

And you are correct that it probably doesn't make sense for the new library structure...
